src/test/resources/solidity/end_to_end_testing_quick/empty.json

```diff
@@ -0,0 +1,13 @@
+{
+  "src/test/resources/solidity/end_to_end_testing_quick/empty.sol:C": {
+    "results": {},
+    "securifyErrors": {
+      "errors": [
+        {
+          "error": "decompilation_error",
```

Comment:
I assume committing this file is intentional, right? What triggers the output `decompilation error`?

Reply:
It's the one solved in #52, so yes it's intentional.